lerobot/configs/default.yaml

```diff
@@ -35,7 +35,7 @@ eval:
   use_async_envs: false
 
 wandb:
-  enable: true
+  enable: false
```

Comment:
@alexander-soare @aliberts: hey guys, please dont do this.

First, I lost a few hours because I wasnt aware of the change as there is no clean warning. It is too big of a change:
<img width="803" alt="Screenshot 2024-05-19 at 23 09 24" src="https://github.com/huggingface/lerobot/assets/4681518/3c239cfa-7cf5-4b42-ad43-56a44ec0c396">

But most importantly, what about the README? You should update it accordingly.
<img width="878" alt="Screenshot 2024-05-19 at 23 12 23" src="https://github.com/huggingface/lerobot/assets/4681518/0325ff1a-8bc6-4d1c-a377-017502437c39">
<img width="878" alt="Screenshot 2024-05-19 at 23 12 40" src="https://github.com/huggingface/lerobot/assets/4681518/99d7719e-cb53-48c3-9d52-05ba0c2ac530">